apollo-router/tests/integration_tests.rs

```diff
@@ -18,8 +19,8 @@ macro_rules! assert_federated_response {
             .query($query)
             .variables(Arc::new(
                 vec![
-                    ("topProductsFirst".to_string(), 2.into()),
-                    ("reviewsForAuthorAuthorId".to_string(), 1.into()),
+                    (ByteString::from("topProductsFirst"), 2.into()),
+                    (ByteString::from("reviewsForAuthorAuthorId"), 1.into()),
```

Comment:
:yarn: the types here look inferred. Using `.into()` here would work for `ByteString` and for `String` at the same time! :sunglasses: 

```suggestion
                    ("topProductsFirst".into(), 2.into()),
                    ("reviewsForAuthorAuthorId".into(), 1.into()),
```